templates/mission_edit.html

```diff
@@ -106,7 +112,7 @@ <h1>Edit Mission</h1>
 					</label>
 				</td>
 				<td>
-					<input type="url" name="wave_url" value="{{ mission.wave_url or '' }}" />
+					<input type="url" name="wave_url" value="{{ mission.wave_url or '' }}" placeholder="https://rizzoma.com/topic/XXXXXXXX...XXXXXXXX" pattern="https://rizzoma.com/topic/[a-z0-9]+" />
```

Comment:
I will add the trailing slash, but it is intentional that it requires a wave link, not a blip link.

Reply:
That's a reasonable requirement. The issue is (at least on chrome, and during my testing) you can be focused on the wave and still have a link to a blip in the URL bar. I think we should have code in `missions.py` to filter the blip off if it is there. If there isn't time for that in this release it can be pushed off and I'll take that issue on in the next release. The site should be as user friendly as possible, we should avoid possible pitfalls with people who are not intimate with Rizzoma URL anatomy and URL structure in general.